.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: goreleaser
+
+on:
+  push:
+    branches:
+      - 'main'
```

Comment:
Do we want it on push to main or only when a tag is made? At first I thought push to main made sense, but now I'm thinking we only want the latter bc goreleaser needs tag?

Reply:
Yeah I was thinking only when a tag is made is probably better, but I'll leave it up to @nguyer